docs/howto/kubernetes/index.md

```diff
@@ -1,40 +1,9 @@
 # Kubernetes in Cleura
 
-<!-- Config interpolation for the company name didn't work for the 
-title here. -->
+In {{brand}} Cloud you can run Kubernetes in several
+ways.
 
```

Comment:
Since you don't _run_ Kubernetes, I would rephrase this sentence as follows: 

In {{brand}} Cloud there are several ways to deploy a Kubernetes cluster.

Reply:
Fair point; I used not quite that but something similar. :)